docs/griptape-framework/drivers/prompt-drivers.md

```diff
@@ -25,6 +25,37 @@ You can pass images to the Driver if the model supports it:
 --8<-- "docs/griptape-framework/drivers/src/prompt_drivers_images.py"
 ```
 
+## Structured Output
+
+Some LLMs provide functionality often referred to as "Structured Output". This means instructing the LLM to output data in a particular format, usually JSON. This can be useful for forcing the LLM to output in a parsable format that can be used by downstream systems.
+
+Structured output can be enabled or disabled for a Prompt Driver by setting the [use_native_structured_output](../../reference/griptape/drivers.md#griptape.drivers.BasePromptDriver.use_native_structured_output).
+
+If `use_native_structured_output=True`, you can change _how_ the output is structured by setting the [native_structured_output_strategy](../../reference/griptape/drivers.md#griptape.drivers.BasePromptDriver.native_structured_output_strategy) to one of:
+
+- `native`: The Driver will use the LLM's structured output functionality provided by the API.
+- `tool`: Griptape will pass a special Tool, [StructuredOutputTool](../../reference/griptape/tools/structured_output_tool.md) and try to force the LLM to use a Tool.
```

Comment:
In what cases would you use the `tool` strategy? Can it result in multiple "tool" calls/outputs for a single LLM call?

Reply:
Actually why do I need to pass the boolean in addition to the strategy name? If I just pass the strategy name is that not sufficient to infer that it should be used? In other words can you just remove the `use_native_structured_output` output flag and rely on `native_structured_output_strategy` exclusively?